### DISCUSSION TITLE
Problem delete titlebar window

### DISCUSSION THREAD

**Discussion by Wasdalt** [AvaloniaUI/Avalonia#16964] [Q&A] [answered]
I am writing a cross-platform application. Where it is necessary that the window opens to all screens of the system. In Windows starting from version 11.1.0, parameters such as stopped working:

 desktop.MainWindow = new MainWindow
 {
     DataContext = new MainWindowViewModel(),
     Width = , // The width of the entire window for all screens, taking into account the scaling factor
     Height = , // The height of the entire window for all screens, taking into account the scaling factor
     Position = new PixelPoint( , ), // The starting point of the entire window for all screens
     Topmost = true,
     ExtendClientAreaToDecorationsHint = true,
     ExtendClientAreaTitleBarHeightHint = 0,
     CanResize = false,
     ExtendClientAreaChromeHints = ExtendClientAreaChromeHints.NoChrome
 }; 

Can you tell me how it is implemented now?

**Comment by timunie** [comment]
Isn't there a property WindowDecorations?

**Reply by Wasdalt** [reply]
There is, but it didn't help

**Reply by Wasdalt** [reply]
![Снимок экрана от 2024-09-08 22-25-57](https://github.com/user-attachments/assets/248e9d4f-b79a-4175-8285-33a73df19e7c)
![Снимок экрана от 2024-09-08 22-30-41](https://github.com/user-attachments/assets/1905fb2c-e01c-4763-ba27-68c20dfff702)

I need a window without a titlebar to be on a full screen. In the first screenshot with my code on version 11.1.0, my window does not want to be on a full screen. In the second screenshot, version 11.0.13 works with my code

**Reply by Wasdalt** [reply, marked as answer]
Width = combinedBounds.Width / mainScreen.PixelDensity,
                Height = combinedBounds.Height / mainScreen.PixelDensity,
                Position = new PixelPoint(combinedBounds.X, combinedBounds.Y),
                WindowState = WindowState.FullScreen,
                Topmost = true,
                ExtendClientAreaToDecorationsHint = true,
                ExtendClientAreaTitleBarHeightHint = -1,
                CanResize = false,
                ExtendClientAreaChromeHints = ExtendClientAreaChromeHints.NoChrome
